src/neural/opencl/clsource/convolve1.opencl

```diff
@@ -107,11 +106,10 @@ __kernel void merge_bn(
     sum += vload_net_t(batch * channels * boardsize * outputs +
 	    (c * boardsize + b) * outputs + o, in);
   }
-  if (means) {
-    const float mean = vload_net_t(o, means);
-    const float scale_stddiv = vload_net_t(o, stddivs);
-    
-    sum = scale_stddiv * (sum - mean);
+  if (biases) {
+    const float bias = vload_net_t(o, biases);
+
+    sum = sum + bias;
```

Comment:
This changes whether relu is defined from being based on existance of means, to existance of biases.
As far as I can tell this has no effect since all convole1's in the net post past and present always have relu - but it could be worth someone double checking.